pkg/haproxy/instance_test.go

```diff
@@ -5148,7 +5172,7 @@ func setupOptions(options testOptions) *testConfig {
 		0,
 		2048,
 	); err != nil {
-		t.Errorf("error parsing responses.lua.tmpl: %v", err)
+		t.Errorf("error parsing responses.http.tmpl: %v", err)
```

Comment:
Is this change needed?

Reply:
Yes looks like this error message was originally copy/pasted from line 5184 without changing the file name to match line 5170.

Reply:
Sure makes sense, messed up the one that this msg is related to.